pom.xml

```diff
@@ -11,8 +11,16 @@
     <groupId>be.xplore</groupId>
     <artifactId>pricescraper</artifactId>
     <version>0.0.1-SNAPSHOT</version>
+    <packaging>pom</packaging>
     <name>pricescraper</name>
     <description>pricescraper</description>
+    <modules>
```

Comment:
the root of the project normally doesnt actually use dependencies,
it should have a dependencymanagement section where you manage dependency versions which you then include in the sub modules